starknet-replay/Cargo.toml

```diff
@@ -29,6 +29,10 @@ pathfinder-common =  { git = "https://github.com/reilabs/pathfinder.git", rev =
 pathfinder-executor =  { git = "https://github.com/reilabs/pathfinder.git", rev = "f802acfdc4e5842fc5f49f8e7ad60ed1379f8bf4" }
 pathfinder-rpc =  { git = "https://github.com/reilabs/pathfinder.git", rev = "f802acfdc4e5842fc5f49f8e7ad60ed1379f8bf4" }
 pathfinder-storage =  { git = "https://github.com/reilabs/pathfinder.git", rev = "f802acfdc4e5842fc5f49f8e7ad60ed1379f8bf4" }
+# `plotters` is using the latest (as of 30-May-2024) commit on the branch
+# `next-release-devel` because it contains the fix for bug #551 related to
+# anchoring of labels when rotated.
```

Comment:
Awesome to see this comment! Do we have a tracking issue for moving back to a stable release?